tokio/src/sync/tests/loom_list.rs

```diff
@@ -8,6 +9,7 @@ macro_rules! if_fuzz {
         $($t)*
     }
 }
+/*
```

Comment:
What's the reason for this?

Reply:
Mistake on my part. Fixed.

Reply:
Oh, we don't need the mod / path hacks anymore.